.tito/custom.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# All rights reserved.
+#
+# Author: Jan Pokorny <jpokorny@redhat.com>
+# (using src/tito/builder/fetch.py from the tito project as a template)
+#
+# This source code is licensed under the GNU Lesser General Public License
+# version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+#
+
```

Comment:
Yes, but we have a style guide, so we should use it :) The style guide's approach is more pythonic, and plays nicely with common python documentation/debugging tools. Also, moving towards consistency in our python style will help people learning the code.